unit-tests/src/test/scala/scala/scalanative/posix/sys/WaitTest.scala

```diff
@@ -0,0 +1,58 @@
+package scala.scalanative.posix.sys
+
+import scala.scalanative.posix.sys.wait._
+import scala.scalanative.posix.unistd.vfork
+import scala.scalanative.unsafe._
+
+import org.junit.Test
+import org.junit.Assert._
+
+class WaitTest {
+  @Test def waitWaitsChildExit(): Unit = {
+    val statLoc = stackalloc[CInt](1)
+
+    def blackHole(a: Any): Unit = ()
+
+    /** these are just here to test the bindings compile and link **/
```

Comment:
This is one of those "gee, doesn't the reviewer have anything better to do"
comments but it becomes relevant when trying to maintain a largest code base over historical time with a sliding cadre of developers.  Not meant to make the review onerous.

I don't think that it is documented in the SN Contributor's Guide but I  think
the current preferred style in Scala Native is for single line comments to 
use the "//" style.  "/**<blank>" may even trigger some Scaladoc(?) issues?

SN is moving towards, but is not yet at, the style described in the Scala.js
style guide. I have to re-read that once a month or so.

@ekrich  If what I am saying, do you think this "preference" is a candidate for the Contributor's Guide, if not already there.

Reply:
done